packages/grafana-data/src/types/theme.ts

```diff
@@ -248,3 +248,7 @@ export interface GrafanaTheme extends GrafanaThemeCommons {
     pageHeader: string;
   };
 }
+
+export interface Themeable {
```

Comment:
Unless we aim to support multiple themes at the same time... maybe this could be a singleton set in `@grafana/ui` -- then we would not need to pass it as props to everyting in @grafana/ui

I think the only downside is that that will make the storybook theme picking a bit more complex

@dprokop -- thoughts?